src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -584,11 +584,11 @@ Copyright (c) .NET Foundation. All rights reserved.
       condition is evaluated.
     -->
     <ItemGroup Condition="'@(RazorGenerate)'!=''">
-      <AllItemsFullPathWithTargetPath Include="@(RazorIntermediateAssembly->'%(FullPath)')">
+      <AllItemsFullPathWithTargetPath Include="@(RazorIntermediateAssembly->'%(FullPath)')" Condition="Exists('%(RazorIntermediateAssembly.FullPath)')">
```

Comment:
@rynowak this was the cause. We include the assembly in the CopyToOutputDirectory item group even when we haven't built (for e.g. if you do a `<ProjectReference Include="..\SimpleMvc\SimpleMvc.csproj" ReferenceOutputAssembly="false" Targets="Restore" />`. Conditionally adding the project files is what we do during `CopyFilesToOutputDirectory`: https://github.com/aspnet/AspNetCore-Tooling/pull/278/files#diff-95bef683fe9f3425f979df61c894d65bR649, so this does not seem incorrect. 

Note that we honor a couple of additional flags (`'$(CopyBuildOutputToOutputDirectory)' == 'true' and '$(SkipCopyBuildProduct)' != 'true')` during `CopyFilesToOutputDirectory`. I'm not changing the code to support just as yet - we'll wait for a user to file an issue.

Reply:
Ignore this. This is possibly an issue, but it's user reported and doing this uncovers some random arcade \ build infrastructure issue that I'd rather not deal with.